crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -42,24 +60,71 @@ pub struct StandardMaterial {
+    /// to get the final "emitting" color of a surface.
+    ///
+    /// This color is multiplied by [`emissive`] to get the final emitted color.
+    /// Meaning that you should set [`emissive`] to [`Color::WHITE`]
+    /// if you want to use the full range of color of the emissive texture.
+    ///
+    /// [`emissive`]: StandardMaterial::emissive
     #[texture(3)]
     #[sampler(4)]
     pub emissive_texture: Option<Handle<Image>>,
-    /// Linear perceptual roughness, clamped to [0.089, 1.0] in the shader
-    /// Defaults to minimum of 0.089
+
+    /// Linear perceptual roughness, clamped to `[0.089, 1.0]` in the shader.
+    ///
+    /// Defaults to minimum of `0.089`.
+    ///
+    /// Low values result in a "glossy" material with specular highlights,
+    /// while values close to `1` result in rough materials.
+    ///
     /// If used together with a roughness/metallic texture, this is factored into the final base
-    /// color as `roughness * roughness_texture_value`
+    /// color as `roughness * roughness_texture_value`.
     pub perceptual_roughness: f32,
-    /// From [0.0, 1.0], dielectric to pure metallic
+
+    /// How "metallic" the material appears, within `[0.0, 1.0]`,
+    /// going from dielectric to pure metallic.
+    ///
+    /// Defaults to `0.01`.
+    ///
+    /// The closer to `1` the value, the more the material will
+    /// reflect light like a metal such as steel or gold.
+    ///
     /// If used together with a roughness/metallic texture, this is factored into the final base
-    /// color as `metallic * metallic_texture_value`
+    /// color as `metallic * metallic_texture_value`.
     pub metallic: f32,
 
+    /// Metallic and roughness maps, stored as a single texture.
+    ///
+    /// The blue channel contains metallic values, and the green channel roughness values.
+    /// Other channels are unused.
+    ///
+    /// Those values are multiplied by the scalar ones of the material; see [`metallic`] and
+    /// [`perceptual_roughness`] for details.
+    ///
+    /// Note that with the default values of [`metallic`] and [`perceptual_roughness`],
+    /// setting this texture has no effect. If you want to exclusively use the
+    /// `metallic_roughness_texture` values for your material, make sure to set [`metallic`]
+    /// and [`perceptual_roughness`] to `1.0`.
+    ///
+    /// [`metallic`]: StandardMaterial::metallic
+    /// [`perceptual_roughness`]: StandardMaterial::perceptual_roughness
     #[texture(5)]
     #[sampler(6)]
     pub metallic_roughness_texture: Option<Handle<Image>>,
-    /// Specular intensity for non-metals on a linear scale of [0.0, 1.0]
-    /// defaults to 0.5 which is mapped to 4% reflectance in the shader
+
+    /// Specular intensity for non-metals on a linear scale of `[0.0, 1.0]`.
+    ///
+    /// Please consider the value as a way to control the intensity of the
+    /// specular highlight of the material, rather than the physical property
+    /// "reflectance."
```

Comment:
```suggestion
    /// The value should be considered as a way to control the intensity of the specular highlight
    /// of the material, i.e. how reflective the material is, rather than the physical property
    /// "reflectance."
```

Reply:
I generally prefer the active voice over the passive voice (per [US government writing recommendations](https://www.plainlanguage.gov/resources/articles/dash-writing-tips/)). I am convinced it makes documentation more readable. Except for the passive voice, your suggestion is an improvement though.